build/tasks/build.js

```diff
@@ -82,13 +89,15 @@ module.exports = function( grunt ) {
 			contents = contents
 				.replace( /define\(\[[^\]]*\]\)[\W\n]+$/, "" );
 		}
+
 		// AMD Name
-		if ( (amdName = grunt.option( "amd" )) != null && /^exports\/amd$/.test( name ) ) {
-			if (amdName) {
+		if ( ( amdName = grunt.option( "amd" ) ) != null && /^exports\/amd$/.test( name ) ) {
```

Comment:
> which was about requiring spaces in function calls

Well, in that discussion Mike suggested to remove all exceptions, so we wouldn't create sophisticated logic to the parentheses rules in JSCS needed to the one preset, that was somewhat the point of that proposal.

> I can just directly ask how you would have treated expression parentheses if JSCS were capable of differentiating them

I would follow the style guide, no matter of it is, i.e. `jquery` preset should represent it.

Personally, i would add more exceptions to that rule as i suggested in jquery/contribute.jquery.org#104  including the "expression parentheses"